draft-ietf-teep-architecture.md

```diff
@@ -944,7 +944,7 @@ of extended claims.
 
 As of the writing of this specification, device and TEE attestations have not been standardized
 across the market. Different devices, manufacturers, and TEEs support different attestation
-algorithms and mechanisms. In order for TEEP to be inclusive, it is agnostic to the format of evidence,
+algorithms and mechanisms (for example, ECDSA-based attestation, Direct Anonymous Attestation (DAA), Enhanced Privacy ID (EPID), a Privacy CA). In order for TEEP to be inclusive, it is agnostic to the format of evidence,
```

Comment:
I think we would need informative references for each of these, to accept this PR

Reply:
I thought about this again and came to the conclusion that this document is not the right place to discuss attestation techniques.